### DISCUSSION TITLE
해당 프로젝트에 적합한 설계는 무엇이라고 생각하시나요?

### DISCUSSION THREAD

**Discussion by wonsss** [woowacourse-study/2022-code-review-study#19] [Lotto] [answered]
크루들 모두 다양한  패턴 또는 자신만의 방법을 사용하여 설계를 하시는 것 같아요.

새로운 도전과 그 시도에서 느끼는 깨달음이 아주 값질 것 같네요! 👍 

그런데 본질로 돌아와서 해당 프로젝트에 적합한 설계는 무엇이라고 생각하시는지 궁금해요.

**Comment by lokba** [comment, marked as answer]
마르코 저도 다른 답변을 드리고 싶지만, 썼다 지웠다 반복해서 
제가 처음부터 쓰고자 한 답변을 적겠습니다.

설계에 정답이 없기에, 적합한 설계는 없다고 생각합니다.🤣

**Comment by moonheekim0118** [comment]
> 아무래도 오버엔지니어링이나 지금 이 구조에서 이게 과연 맞을까?에 대해서 고민이 많으실 거 같아요! 걍 말씀드리자면ㅋ 웬만한 패턴 다 안맞을거에요ㅎㅎ; 지금도 앞으로도 비교적 작은 프로젝트일테니까요..ㅎ 다만 지금은 학습과정에서 이것저것 해보면서 느껴보는 게 중요하다고 생각해요. 지금 MVC, 추상화 등을 해보면서 호프가 느껴야할 점은 일단 '일단 지금같이 작은 프로젝트에서는 좀 맞지 않네ㅎ.., 이 프로젝트가 ~~~~~방향으로 발전하면 이렇게 쓰이려나? 쓰이겠구나 그러면 이 부분이 유효하게 작용할 거 같다.' 정도..이지 않을까 싶습니다!
